back/engines/commercial/analysis/spec/acceptance/analyses_spec.rb

```diff
@@ -124,6 +125,26 @@
     end
   end
 
+  patch 'web_api/v1/analyses/:id' do
+    with_options scope: :analysis do
+      parameter :show_insights, 'Whether to show insights or not', required: false
```

Comment:
As this was also included in the create params we probably also want to add this documentation there.